extensions/control-plane/store/sql/contract-negotiation-store-sql/src/test/java/org/eclipse/edc/connector/store/sql/contractnegotiation/SqlContractNegotiationStoreExtensionTest.java

```diff
@@ -40,18 +45,25 @@ class SqlContractNegotiationStoreExtensionTest {
 
     @Test
     void initialize(ServiceExtensionContext context, ObjectFactory factory) {
+        var ctx = spy(context);
+        var config = mock(Config.class);
+        when(ctx.getConfig()).thenReturn(config);
+        when(config.getString(DATASOURCE_NAME_SETTING, DataSourceRegistry.DEFAULT_DATASOURCE)).thenReturn("test");
+
         context.registerService(DataSourceRegistry.class, mock(DataSourceRegistry.class));
         context.registerService(TransactionContext.class, mock(TransactionContext.class));
         context.registerService(ContractNegotiationStatements.class, null);
         context.registerService(TypeManager.class, new TypeManager());
 
         extension = factory.constructInstance(SqlContractNegotiationStoreExtension.class);
```

Comment:
Also not needed, the `SqlContractNegotiationStoreExtension` can be injected as method parameter. (same for other tests)

Reply:
I think in this case it's still needed for custom injected services in both test right?